tests/unittest/auto_deploy/singlegpu/shim/test_cached_sequence_interface.py

```
@@ -1109,6 +1202,72 @@ def test_generic_state_handler_allocated_locally(paged_kv_cache_config):
+    )
+
+    interface.add_resource(
+        "kv_cache_0", KVPagedResourceHandler(8, 64, dtype=torch.float16, attention_type="mha")
+    )
+    interface.add_resource(
+        "kv_cache_1", KVPagedResourceHandler(8, 80, dtype=torch.float16, attention_type="mha")
+    )
+
+    with pytest.raises(
+        RuntimeError,
+        match="reject_unmanaged_persistent_caches.*KVPagedResourceHandler",
```

Ruff (0.15.13)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)